tests/integration/test_cases/summary.py

```
@@ -148,4 +279,678 @@ async def _test_fetch_summary(self) -> bool:
+                update_url = f"{self.ingestor_server_url}/v1/documents"
+
+                # Test 1: Multiple ranges with first 5 and last 5 pages (tests positive + negative ranges)
+                page_filter = [[1, 5], [-5, -1]]
+
+                logger.info(
+                    "📤 Testing filter: first 5 pages [1, 5] and last 5 pages [-5, -1]"
+                )
+
+                # Prepare FormData with file and JSON data
+                data = aiohttp.FormData()
+                data.add_field("documents", open(file_path, "rb"), filename=filename)
```

Ruff (0.14.5)

Async functions should not open files with blocking methods like `open`

(ASYNC230)